packages/app/server/utils/octokit.ts

```diff
@@ -21,7 +24,7 @@ export function useOctokitApp(event: H3Event): AppType {
       privateKey,
       webhooks: { secret: webhookSecret },
       Octokit: Octokit.defaults({
-        baseUrl: ghBaseUrl,
+        ...(opts?.ignoreBaseUrl ? {} : { baseUrl: ghBaseUrl }),
```

Comment:
why do we need this?

Reply:
i mean there should be an issue with your env variables if it does not work, since it's used by both the prod and staging.